doc/maintenance.md

```diff
@@ -231,7 +231,7 @@ publishing the actual release notes at {doc}`changes`.
 
 ### Check the README Syntax
 
-GitHub is a bit forgiving when it comes to the RST syntax in the README but PyPI is not.
+GitHub is a bit forgiving when it comes to the syntax in the README but PyPI is not.
 To check the README syntax, visit the
 [PyGMT TestPyPI release history](https://test.pypi.org/project/pygmt/#history), select
 the latest commit, and review the left sidebar and project description for any errors.
```

Comment:
https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#creating-a-readme-file

PyPI uses the [GitHub Flavored Markdown](https://github.github.com/gfm/) syntax by default, so we no longer need to check the README syntax on PyPI.

Reply:
Done in 13426785a1985df61496ec1286cf80b2da3f9298.